src/cksum/cksum.rs

```diff
@@ -51,12 +52,12 @@ fn cksum(fname: &str) -> IoResult<(u32, usize)> {
     let mut file_buf;
     let rd = match fname {
         "-" => {
-            stdin_buf = stdin_raw();
-            &mut stdin_buf as &mut Reader
+            stdin_buf = stdin();
```

Comment:
It might be somewhat more efficient to do buffered reads.

To do that, you could import `std::io::BufReader`, and then wrap the `stdin()` call, like `BufReader::new(stdin())`.